modules/kubernetes-addons/variables.tf

```diff
@@ -96,6 +96,12 @@ variable "cluster_autoscaler_helm_config" {
 }
 
 #-----------Amazon Managed Service for Prometheus-------------
+variable "enable_amp_for_prometheus" {
```

Comment:
We don't want to use `amp` right? So probably `enable_amazon_prometheus` is a better prop name. Matches the naming convention of the exported values as well.

Reply:
will change that